parsec/core/gui/workspaces_widget.py

```diff
@@ -618,8 +676,8 @@ def reencrypt_workspace(
         if r != _("ACTION_WORKSPACE_REENCRYPTION_CONFIRM"):
             return
 
-        @contextmanager
-        def _handle_fs_errors():
+        @contextmanager  # type: ignore[arg-type]
```

Comment:
Why ignore ?

Reply:
This is not needed anymore with a return type of `Iterator[None]`